### DISCUSSION TITLE
how to open the offset menu to change the position of your character in 1.6.1?

### DISCUSSION THREAD

**Discussion by HePpYPro** [KadeArchive/Kade-Engine#1730] [Q&A] [answered]
I'm making a mod and i want to change his position (specificlly the y one) and i don't know how to change to the OFFSET MENU NOT THE CHARTING MENU. So i would appriciate the help and thank you

**Comment by EcoVeni** [comment]
try pressing "8" on your keyboard

**Reply by HePpYPro** [reply]
Yeah but that's the problem. When I press it, it does nothing so

**Reply by EcoVeni** [reply]
did u do "lime test windows -debug" ?

**Comment by isakube** [comment]
build in debug mod and press 8

**Comment by HePpYPro** [comment]
But for me it doesn't work or idk

**Comment by BoloVEVO** [comment, marked as answer]
lime test windows -debug
Press '0' to edit boyfriend offset
Press '6' to edit opponnent offset.

**Reply by HePpYPro** [reply]
Thank you, I've discovered that a few days ago, but still thank you a lot
